### ISSUE TITLE
`ftplugin` has no effect

### ISSUE THREAD

**Issue by AlexvZyl** [rcarriga/nvim-dap-ui#283]
I have the following inside `~/.config/nvim/after/ftplugin/dapui_stacks.lua`:
```lua
vim.cmd "setlocal winbar=\\ Stacks"
vim.cmd "setlocal statuscolumn=\\ "
```

However, this does not seem to do anything.  However, when I put the following inside `~/.config/nvim/after/ftplugin/dap-repl.lua`:
```lua
vim.cmd "setlocal winbar=\\ REPL"
vim.cmd "setlocal statuscolumn=\\ "
```

It works.

![image](https://github.com/rcarriga/nvim-dap-ui/assets/81622310/7402d357-8e12-4683-a200-837182ef202d)

**Comment by phortonssf** [comment]
Off topic but how did you get the dap debug ui buttons,  continue, setup over etc... in the top right. Nice them too which one is that?

**Comment by AlexvZyl** [comment]
The theme is [nordic.nvim](https://github.com/AlexvZyl/nordic.nvim) :)

Check out [these lines](https://github.com/AlexvZyl/nvim/blob/eab5de8ab396147ec2d38974a22da35f2312f02d/lua/alex/lang/debugger/ui.lua#L67-L79) for the controls.

**Comment by tmillr** [comment]
I don't think it has anything to do with `ftplugin`, rather window options in general. It seems this plugin is constantly setting/re-initializing the window opts, which makes it very hard to customize or otherwise set them yourself (e.g. `winbar`).
